pages/your_domain/microbial_biotechnology.md

```diff
@@ -12,27 +12,143 @@ In the event that no adequate Problem page exists for a problem that can be gene
+
+Current data standards to capture this level of taxonomic and phenotypic data are still evolving, with notable work on the [Access to Biological Collection Data Schema](https://www.tdwg.org/standards/abcd/) and the activities of the [biodiversity information standards task force (TDWG)](https://www.tdwg.org/). The Darwin Core standard from the [TDWG](https://www.tdwg.org/) is an appropriate standard to provide metadata about the taxonomic properties of a particular microorganism. The [NCBI taxonomy homepage](https://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/) can also provide appropriate taxon IDs for recording taxonomic information. Information about proposed standardised nomenclature for prokaryotes can be found at the [List of Prokaryotic names with Standing in Nomenclature (LPSN)](https://lpsn.dsmz.de/) (Parte et al., 2020).
+
+For sharing information, databases such as the [Bacterial Diversity Metadatabase (Bacdive)](https://bacdive.dsmz.de), have arisen. Of course users can also deposit their strains and associated information in strain repository such as the [National Collection of Industrial, Food and Marine Bacteria (NCIMB)](https://www.ncimb.com/culture-collection/) or the [American Type Culture Collection (ATCC)](https://www.lgcstandards-atcc.org/?geo_country=gb) . There are also many organisations established for individual species of microorganisms, the [Bacillus Genetic Stock Centre (BGSC)](http://www.bgsc.org/) being one example. 
+ 
+Data standards for recording the information about where a microorganism was isolated from do exist and are also covered in other RDM guides such as the [marine metagenomics](https://rdmkit.elixir-europe.org/marine_metagenomics.html) RDM guide.  Information can also be found in a publication by Ten Hoopen and colleagues (Ten Hoopen et al., 2015). [The Environment Ontology](https://sites.google.com/site/environmentontology/) is also relevant here. 
+
+The genetic properties of a host is typically specified as part of a study where the host is modified or used to harbour a genetic construct. The description of the host can therefore be expanded from the taxonomic information to information about which genes have been inserted into the host genome, which genes have been mutated and the parental information about the strain. A set of genetic nomenclature standards have been established by microbiologists and have been used for many years. These are still a useful way of communicating data about the genotype of a strain, see -  (Maloy and Hughes, 2007).  Databases such as [CellRepo](https://cellrepo.herokuapp.com/) allow strains that have been barcoded to be tracked using a version control type system (Tellechea-Luzardo et al., 2020).
```

Comment:
This is very detailed. Some of the text is what can be moved to the "consideration" section.